rowan/components/Deployment.ston

```diff
@@ -20,7 +21,7 @@ RwSimpleProjectLoadComponentV2 {
           },
           'Buoy-Collections-GS64-Extensions' : {
             'symbolDictName' : 'Globals'
-          },
+           },
```

Comment:
You're missing here the mapping to Globals. Given that 'Buoy-Exception-Handling' extends BlockClosure and it's declared in the Globals symbol list.